Library/Homebrew/keg_relocate.rb

```diff
@@ -75,8 +78,8 @@ def text_files
       path.find do |pn|
         next if pn.symlink? || pn.directory?
         next if Metafiles::EXTENSIONS.include? pn.extname
-        if Utils.popen_read("/usr/bin/file", "--brief", pn).include?("text") ||
-           pn.text_executable?
+        if pn.text_executable? ||
+           Utils.popen_read("/usr/bin/file", "--brief", pn).include?("text")
```

Comment:
I tried to get rid of https://github.com/Homebrew/brew/blob/c9c7f462d37500549127efba96c7a25e5c70de4a/Library/Homebrew/cask/lib/hbc/artifact/uninstall_base.rb#L11 before by using `getconf ARG_MAX`, but somehow got side-tracked. 😄